app/presenters/document_presenter.rb

```diff
@@ -129,22 +125,34 @@ def default_date_metadata
     caption = first_edition? ? "Published" : "Updated"
 
     {
-      caption => published_at,
+      caption.downcase => {
```

Comment:
Can't we just make the values of caption lower case in the first place, since they're not used for anything else?